portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase

```diff
@@ -490,4 +490,37 @@ definition {
 		}
 	}
 
+	@description = "Add a new admin user, the user creates a new page and web content, and adds a web content widget to the page"
```

Comment:
Suggestion for the description, you'd better simplify it, you can begin with "`Assert ...`".; For the test name, you'd better with "Can", for example, `NewAdminUserCanAddPageAndWebContent`